build/AzureInfo.props

```diff
@@ -1,27 +1,36 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <Product>Sdk</Product>
+    <IsOrchestratedPublish Condition="'$(PB_PublishBlobFeedUrl)' != ''">True</IsOrchestratedPublish>
```

Comment:
Should this be IsOrchestratedBuild? That's how we usually refer to it.

Reply:
Are we open to the chance that we can have an orchestrated build with a regular publish - or is that never going to happen?

Reply:
@johnbeisner Probably not, but...possible?

Reply:
That's actually the primary reason for the PB_PublishType flags enum.  We can turn on default publishing in an orchestrated build